libbeat/.gitignore

```diff
@@ -8,6 +8,7 @@ _output
 /docs/html_docs
 .vagrant
 build
+/dashboards/import_dashboards
```

Comment:
`import_dashboards` is the exec, built from `import_dashboards.go`. This is generated with `make build-dashboards`.